portal/src/views/Docs/Docs.vue

```diff
@@ -121,89 +83,93 @@ interface RouteDef {
 	description: string;
 	method: string;
 	path: string;
-	params: swag.Operation;
+	params: swag.Spec; // changed to swag.Spec because it fit our needs better than swag.Operation
```

Comment:
Can you comment above this why swag spec fits out needs better than swag operation.
So essentially explain why you believe that swag.Spec is required here. or what it does?